webapi/src/main/scala/org/knora/webapi/routing/RouteUtilZ.scala

```diff
@@ -0,0 +1,31 @@
+package org.knora.webapi.routing
+import zio.Task
+import zio.ZIO
+
+import java.net.URLDecoder
+
+import dsp.errors.BadRequestException
+
+object RouteUtilZ {
+
+  /**
+   * Url decode a [[String]].
+   * Fails if String is not a well formed utf-8 [[String]] in `application/x-www-form-urlencoded` MIME format.
+   *
+   * Wraps Java's [[java.net.URLDecoder#decode(java.lang.String, java.lang.String)]] into the zio world.
+   *
```

Comment:
arguably, this is an implementation detail which is not necessarily needed in the docstring. IMO, docstring should contain the "what" not the "how"